modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -763,6 +764,11 @@ const register = (editor: Editor): void => {
     default: (_ctx: PatternTypes.DynamicPatternContext): PatternTypes.Pattern[] => [ ]
   });
 
+  registerOption('text_patterns_block_pre_execution', {
```

Comment:
```suggestion
  registerOption('text_patterns_on_space', {
```

I'm bad at names too. We need more ideas.